gradle-palantir-java-format/build.gradle

```diff
@@ -21,75 +25,65 @@ dependencies {
     testImplementation 'org.assertj:assertj-core'
     testImplementation project(':palantir-java-format')
 
-    // Have to configure this manually in order to resolve the implicitDependencies together with runtimeClasspath
-    // This is to allow both our code and spotless to end up on the same class loader in tests.
-    // Ordinarily when running, these would end up on the same classloader
-    pluginClasspath configurations.implicitDependencies
-    pluginClasspath configurations.runtimeClasspath
-    pluginClasspath sourceSets.main.output
-    compileOnly configurations.implicitDependencies
-    implicitDependencies 'com.diffplug.spotless:spotless-plugin-gradle'
+    pluginClasspath 'com.diffplug.spotless:spotless-plugin-gradle'
```

Comment:
If you'd like to only see `com.diffplug.spotless:spotless-plugin-gradle` only defined once, a quick way to achieve this is by modifying the `pluginClasspath` configuration above to have an `extendsFrom compileOnly`.